# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/all.js
# Commit: 258c6c199656
# Full Hash: 258c6c1996568b3e7d3ca442a2d87df3f60a4b32
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-07 16:20:31
# Regressor Bug: 1464032
# File Overlap Count: 1
# Description:
#   Bug 1464032 Part 6: Add remote canvas pref and refactor TextuteData creation to use it. r=mattwoodrow
#   
#   This is ground work for when we will be returning a recording TextureData for
#   certain types in subsequent patches.
# ==============================================================================

diff -r c83dbcc4dade -r 258c6c199656 modules/libpref/init/all.js
--- a/modules/libpref/init/all.js	Sun Dec 02 14:14:19 2018 +0000
+++ b/modules/libpref/init/all.js	Wed Nov 28 20:44:27 2018 +0000
@@ -839,6 +839,7 @@
 // e.g., pref("gfx.canvas.azure.backends", "direct2d,skia,cairo");
 pref("gfx.canvas.azure.backends", "direct2d1.1,skia,cairo");
 pref("gfx.content.azure.backends", "direct2d1.1,skia,cairo");
+pref("gfx.canvas.remote", false);
 #else
 #ifdef XP_MACOSX
 pref("gfx.content.azure.backends", "skia");
